src/main/java/com/example/htmlpractice/AddressController.java

```diff
@@ -2,29 +2,37 @@
 
 import com.example.htmlpractice.model.Address;
 import com.example.htmlpractice.model.AddressData;
-import jakarta.servlet.http.HttpSession;
+import com.example.htmlpractice.model.ZipcodeFrom;
```

Comment:
英語だと
ZIP Codeと2語みたいなので
```suggestion
import com.example.htmlpractice.model.ZipCodeFrom;
```
がよいかもです。

Reply:
c7dd86d00eaa60f2960d0981214b7d8d4bc6611d
ZipCodeFormクラスへと変更しました